subsys/usb/usb_device.c

```diff
@@ -1600,17 +1608,20 @@ int usb_enable(void)
 	/* Enable control EP */
 	ret = usb_dc_ep_enable(USB_CONTROL_OUT_EP0);
 	if (ret < 0) {
-		return ret;
+		goto usb_enable_error;
 	}
 
 	ret = usb_dc_ep_enable(USB_CONTROL_IN_EP0);
 	if (ret < 0) {
-		return ret;
+		goto usb_enable_error;
 	}
 
 	usb_dev.enabled = true;
 
-	return 0;
+usb_enable_error:
```

Comment:
I would like this to be renamed to `out` as it is the exit path for success as well.